packages/framework/src/Resources/config/parameters_common.yml

```diff
@@ -0,0 +1,21 @@
+parameters:
+    # Default Redis keys prefix
+    build-version: '0000000000000000'
+
+    # Seed for demo data generation
+    faker.seed: 1234
+
+    # Use Shopsys validator factory for fp/jsformvalidator-bundle
+    fp_js_form_validator.factory.class: Shopsys\FrameworkBundle\Form\JsFormValidatorFactory
```

Comment:
I think that overriding of class parameters like this should stay in services.yml, because It is part for building DI container and their services...

It is my point of view...

Reply:
A fair point. You're right it's more like a "service" thing. 
I'm not sure why it needs to be adjusted via parameters... seems a little smelly to me, but we still can't do anything about it. 

I give it a second thought. Thanks for the opinion 👍

Reply:
I moved all three parameters with overriding of class back to the `services.yml` as it really makes much more sense.